doc/reference/power_management/index.rst

```diff
@@ -431,7 +431,58 @@ later calling :c:func:`pm_device_wakeup_enable`.
    It is responsability of driver or the application to do any additional
    configuration required by the device to support it.
 
+Device Power Management Policies
+================================
 
+The power management subsystem supports the following device power
+management policies:
+
+* Default
+* Application
+
+The device policy manager is responsible to inform the power subsystem
+which device power state the system should use for a device based on
+the system power state.
+
+When the system is changing its power state it iterates over available
+devices and asks the policy manager which state the device should be
+set based on the system power state. The policy is defined by
+:c:func:`pm_device_policy_next_state`.
+
+Default
+-------
+
+The default device power management policy is a hardcoded map between
+system power states and device power states. The relation between these
+states is:
+
+.. code-block:: c
+
+   PM_STATE_ACTIVE          -> PM_DEVICE_STATE_ACTIVE
+   PM_STATE_RUNTIME_IDLE    -> PM_DEVICE_STATE_LOW_POWER
+   PM_STATE_SUSPEND_TO_IDLE -> PM_DEVICE_STATE_LOW_POWER
+   PM_STATE_STANDBY         -> PM_DEVICE_STATE_LOW_POWER
```

Comment:
`PM_DEVICE_STATE_LOW_POWER` is not used by any in-tree driver anymore. It's there for compatibility until its meaning/usage is clear, so I'd not design new facilities on top of it.

Reply:
That makes sense, if no driver is using it maybe it is better to just map to suspend. I did this way to keep the same behavior we currently have:
```
	case PM_STATE_RUNTIME_IDLE:
		__fallthrough;
	case PM_STATE_SUSPEND_TO_IDLE:
		__fallthrough;
	case PM_STATE_STANDBY:
		/* low power peripherals. */
		if (pm_low_power_devices()) {
			SYS_PORT_TRACING_FUNC_EXIT(pm, system_suspend,
					ticks, _handle_device_abort(z_power_state));
			return _handle_device_abort(z_power_state);
		}
		break;
```

I think for this pr would be better to keep the same behavior and then we can change it after this, I prefer to do it step by step. This default policy is just reproducing the current behavior, but it allows in future we define device power states (including transitions) in device tree and using this info in the policy without need to change the rest of the subsystem.

Reply:
what do you think ?

Reply:
I think we should first clarify states and avoid building more infrastructure on top of something that is, IMO, not clear on how it is supposed to be used.